docs/Using-Push-Service.md

```diff
@@ -56,16 +56,21 @@ __Optional parameters:__
 All available methods of the `IPushService` API are:
 
 - `acceptLanguage` - Language settings, that will be sent along with each request.
-- `register(fcmToken: String, listener: IPushRegisterListener)` - Registers Firebase Cloud Messaging token on the backend
-    - `fcmToken` - Firebase Cloud Messaging token.
-    - `listener` - Called when the request finishes.
+- `register(fcmToken: String, callback: (result: Result<Unit>) -> Unit)` - Registers Firebase Cloud 
+- `registerHuawei(hmsToken: String, callback: (result: Result<Unit>)` -> Unit)
 
-## Registering to Push Notifications
+Messaging token on the backend
+
+- `fcmToken` - Firebase Cloud Messaging token.
+- `hmsToken` - Huawei Mobile Services token
+- `callback` - Called when the request finishes.
 
+## Registering to Push Notifications
+### Android
```

Comment:
Well, Huawei is also Android, but without Google Services. So this should be probably "Android (with Google Play Services)"